packages/documentation/src/components/dashboard/DashboardDataFetch.jsx

```diff
@@ -64,3 +71,55 @@ export async function crossAppImportDataFetch(repo) {
   }
   return crossAppImportDataResponse.json();
 }
+
+export async function deploysFetch(
```

Comment:
This looks great! Really easy to reference each commit from this to get the status